src/apps/competitions/models.py

```diff
@@ -85,9 +85,9 @@ def save(self, force_insert=False, force_update=False, using=None,
 
 
 class CompetitionParticipant(models.Model):
-    user = models.OneToOneField(settings.AUTH_USER_MODEL, null=False, blank=False, related_name='participant',
-                                on_delete=models.DO_NOTHING)
-    competition = models.OneToOneField(Competition, related_name='participants', on_delete=models.CASCADE)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, null=False, blank=False, related_name='participants',
```

Comment:
Should we set this to nullable and make it `models.SET_NULL`..? trying to think this through... in the future we'll want to be able to completely delete a user, but maybe we'll want their participant/submissions around?